spec/diffux_ci_spec.rb

```diff
@@ -523,4 +523,74 @@ def snapshot_file_exists?(description, size, file_name)
         .to eq(false)
     end
   end
+
+  describe 'when additional files are served from public directories' do
+    before do
+      @tmp_pub_dir = File.join(@tmp_dir, 'public')
```

Comment:
Since this variable isn't used outside of this block, you don't need the `@`. If you remove it, it will make your code a little clearer.